### PR TITLE
Add new Adapter rdnBidAdapter

### PR DESCRIPTION
**Pull request by iijimalo1023**
## Type of change
- [x] New bidder adapter

## Description of change
- test parameters for validating bids
```
{
  bidder: 'rdn',
  params: {
    adSpotId:10000
  }
}
```
The corresponding size of this test parameter is here.
width:300 height:250

- contact email of the adapter’s maintainer
engineer@lob-inc.com

- [x] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

When the pull request is merged, the document can be referred to below.
https://github.com/prebid/prebid.github.io/pull/964
https://github.com/prebid/prebid.github.io/tree/master/dev-docs/bidders/rdn.md

### CODE REVIEW

File: test/spec/modules/rdnBidAdapter_spec.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
+import { expect } from 'chai'
+import * as utils from 'src/utils'
+import { spec } from 'modules/rdnBidAdapter'
+import { newBidder } from 'src/adapters/bidderFactory'
+
+describe('rdnBidAdapter', function() {
+  const adapter = newBidder(spec)
+  const ENDPOINT = 'https://s-bid.rmp.rakuten.co.jp/h'
+
+  describe('inherited functions', () => {
```
Comment (jsnellbaker) [inline test/spec/modules/rdnBidAdapter_spec.js]: Could you update the various `, () => {` arrow references used in this file to use the `, function() {` style instead?

In a recent project-wide update, we converted all these arrow references in the various `spec.js` files to the other style to be more in-line with Mocha.  So we'd like to have new adapters maintain this consistency in their spec files.

See https://github.com/prebid/Prebid.js/issues/2987 for additional context.

**Comment by jsnellbaker** [comment]
In addition - it seems like the unit tests have a code-coverage rate of 62.96% (you can check this by running `gulp test-coverage` and `gulp view-coverage` in your Prebid.js project).

Ideally we'd like to have adapter's unit tests cover at least 80% or more.  

The biggest gap seems to be the `getUserSyncs` area of the adapter code.  If you do rewrite some of it (based on the feedback noted earlier), it may lessen the gap.  If not, could you look into adding an additional unit test for this area?

File: modules/rdnBidAdapter.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
+import { registerBidder } from 'src/adapters/bidderFactory'
+import * as utils from 'src/utils'
+import { BANNER } from 'src/mediaTypes'
+
+const BIDDER_CODE = 'rdn'
+const ENDPOINT = 'https://s-bid.rmp.rakuten.co.jp/h'
+
+export const spec = {
+  code: BIDDER_CODE,
+  aliases: ['rdn'],
```
[the remark this answers is not in the document]
Comment (iijimalo1023) [reply modules/rdnBidAdapter.js]: I deleted it because it does not use it.

**Comment by iijimalo1023** [comment]
Hi @jsnellbaker 

Thank you for confirmation. We will contact you regarding the items pointed out below.

As a result of using the parameter of the md file, there was an error in the parameter regarding the problem where the status code 204 was returned. Since we made a fix here and pushed, please check again.

In addition, we added a test code concerning the code-coverage rate shortage of the test code. I confirmed that it is 96.3% in Statements.

Sorry to trouble you, but please confirm again.

**Comment by jsnellbaker** [comment]
@iijimalo1023 Thanks for making the additional updates; everything looks good now.

**Pull request merged**